app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
```

Comment:
add blank line between different import groups

combine imports of errors into one